pkg/controllers/cloud/cloudentityselector_controller.go

```diff
@@ -47,6 +44,9 @@ type CloudEntitySelectorReconciler struct {
 	Scheme               *runtime.Scheme
 	selectorToAccountMap map[types.NamespacedName]types.NamespacedName
 	Poller               *Poller
+	pendingSyncCount     int
+	// indicates whether a controller can reconcile CRs.
```

Comment:
```suggestion
	// indicates whether a controller can reconcile CRs.
```
indicates whether controller has reconciled all CRs